app.js

```diff
@@ -5,71 +5,116 @@ alert('Okay, ' + userName + ', my name is Sean.');
+if (myFacialHair.toLowerCase() === 'yes' || myFacialHair.toLowerCase() === 'y') {
+  alert('You are good at this, ' + userName);
+} else if (myFacialHair.toLowerCase() === 'no' || myFacialHair.toLowerCase () === 'n') {
   alert('I\'m trying to grow an impressive beard, you couldn\'t tell? Wrong, just wrong, ' + userName);
-} else if (myHeight !==true){
+} else if (myFacialHair !== true){
   alert('Seriously? Yes or no questions, ' + userName);
-  console.log(userName + ' answered "Do I have facial hair?" with ' + myFacialHair);
 }
+console.log(userName + ' answered "Do I have facial hair?" with ' + myFacialHair);
+
 var myShirt = prompt ('Final question: Did I wear a blue shirt on Monday?');
-  if (myHeight.toLowerCase() === 'yes' || myHeight.toLowerCase() === 'y') {
-  alert('Good guess! You\'re right! ' +userName);
-} else if (myHeight.toLowerCase() === 'no' || myHeight.toLowerCase () === 'n') {
+if (myShirt.toLowerCase() === 'yes' || myShirt.toLowerCase() === 'y') {
+  alert('Good guess! You\'re right! ' + userName);
+} else if (myShirt.toLowerCase() === 'no' || myShirt.toLowerCase () === 'n') {
   alert('You\'re right! Sike!');
-} else if (myHeight !==true){
+} else if (myShirt !== true){
   alert('Okay, I\'m done with you, ' + userName);
-  console.log(userName + ' answered "Finally question: Did I wear a a blue shirt on Monday?" with ' + myShirt);
 }
-confirm('Thanks for playing! Now you can read a little bit more about me.');
+console.log(userName + ' answered "Finally question: Did I wear a a blue shirt on Monday?" with ' + myShirt);
+
+var anotherGame = prompt ('Want to play another guessing game?');
+if (anotherGame.toLowerCase() === 'yes' || anotherGame.toLowerCase() === 'y') {
+  alert('Okay, let\'s do this');
+} else if (anotherGame.toLowerCase() === 'no' || anotherGame.toLowerCase() === 'no') {
+  alert('No? You Sure?');
+} else if (anotherGame !== true){
+  alert('You really don\'t like answering with a yes or no, do you? Well, you don\'t have a choice ', + userName);
+}
+console.log(userName + ' answered "Want to play another guessing game?" with ' + anotherGame);
+
+var computerChoice = Math.floor((Math.random() * 10) + 1);
+var userGuess = 1;
+while (userGuess <= 4) {
+  var userChoice = prompt('Guess my number. Pick a number between 1-10');
+  userGuess++;
+  if (userChoice == computerChoice) {
+    alert('You\'re correct!');
+    break;
+  } else if (userChoice < computerChoice) {
+    alert('You guessed lower than my number, try again!');
+  } else if (userChoice > computerChoice) {
+    alert('You guessed higher than my number, try again!');
+  } else {
+    alert('You lose!');
+  }
+}
+
+var myAnswers = ['band of brothers', 'casablanca', 'fences', 'arrival','hacksaw ridge','passengers','dr. strange','finding dora','moana','john wick'];
+for (i = 1; i < 7; i++){
+  var userAnswer = prompt('Can you guess one of my favorite movies? Try: ' + i).toLowerCase();
+  if( myAnswers.indexOf(userAnswer) != -1) {
```

Comment:
this works and it isn't a big deal because that method always returns a number, but I want you to use strict comparisons whenever possible.  next time use the strict inequality comparison operator: `!==`